### ISSUE TITLE
Missing Italic for Slashed Zero

### ISSUE THREAD

**Issue by Typas** [cormullion/juliamono#138]
Hi! Thanks for visiting.

- found a serious mistake/error? Report it and I'll try to fix it sometime.

The "italic" version of slashed zero is not italic. It remains the same as its normal form.

The result below is produced by XeLaTeX with minted package

```latex
\begin{minted}{rust}
struct Struct0123;
//    comment0123
   println!("0123");
\end{minted}
```

The default zero, matches other italic numbers
```latex
\setmonofont[StylisticSet={1, 3, 5, 6, 8}]{JuliaMono}
```
![Screen Shot 2022-04-02 at 6 06 10 PM](https://user-images.githubusercontent.com/44962044/161378416-d8a86bd7-2ac5-45ed-90d5-e48303e0c2d7.png)

Slashed Zero version, remains normal
```latex
\setmonofont[StylisticSet={1, 3, 5, 6, 8}, Numbers=SlashedZero]{JuliaMono}
```
![Screen Shot 2022-04-02 at 6 07 33 PM](https://user-images.githubusercontent.com/44962044/161378428-b90931c4-6833-4f47-bc07-da1fbe9c9737.png)

**Comment by cormullion** [comment]
Hi - thanks for spotting that! Will fix soon... :)

**Comment by cormullion** [comment]
<img width="216" alt="Screenshot 2022-09-15 at 11 00 49" src="https://user-images.githubusercontent.com/52289/190375946-5df16d1e-2ce7-4744-a549-af0aa4f1512a.png">

hopefully fixed in 0.046
